integrations/extensions/docs/elasticsearch-install-and-setup/python-document-ingestion/doc-ingestion.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
I think it would be easier to use the guide if we convert this notebook into a Python script IMO, since we need to run some command lines anyway before starting the notebook. 

Another option is to put most of the commands into the notebook, which itself is like documentation. I personally prefer the first option.

Reply:
IMO, using a notebook may be better because it allows you to execute things separately . For example, you may have processed all documents into chunks but may run into an elastic error so you have the ability to run just the ingestion separately. 

Another example is, it allows one to modify parameters of the bulk api if needed separately from the chunking without having to re-run the whole script again especially if there's a lot of docs to be processed

Reply:
That's fair. If a user is going to use this guide as a starter kit and will need to customize it or debug issues with it, then a notebook should be better.